consensus/reactor.go

```diff
@@ -516,6 +568,130 @@ OUTER_LOOP:
 	}
 }
 
+func (conR *ConsensusReactor) queryMaj23Routine(peer *p2p.Peer, ps *PeerState) {
```

Comment:
Why do this with a dedicated go-routine and sleeps and not via an event triggered broadcast, like we do with broadcastNewRoundStep and broadcastHasVoteMessage ?